packages/logs/src/domain/configuration.ts

```diff
@@ -30,6 +32,6 @@ export function validateAndBuildLogsConfiguration(
     /**
      * arbitrary value, byte precision not needed
      */
-    requestErrorResponseLengthLimit: 32 * ONE_KILO_BYTE,
+    requestErrorResponseLengthLimit: DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏the comment should be with the constant then